app/modules/fcm/controllers/cytoscape.js

```diff
@@ -309,6 +310,16 @@ angular.module('pcApp.fcm.controllers.cytoscapes', [])
             return input;
         };
 
+        $scope.getUserId = function () {
+            var userId = '';
+            if ($scope.user != null || $scope.user != '') {
+                var userPathArray = $scope.user.state.userPath.split('/');
+                if (userPathArray.length > 2)
+                    userId = parseInt(userPathArray[userPathArray.length - 2], 10);
+            }
+            return userId;
+        };
+
```

Comment:
@mohammedbahja Why are you doing this? You can just store the entire path. Besides this will most likely not work, since Auth is resolved async and you are trying to access it immediately.